Dockerfile.server

```diff
@@ -1,28 +1,30 @@
-FROM node:16-bullseye
+### Build stage
+FROM node:16-bullseye AS builder
 
 RUN apt-get update \
     && apt-get install -y \
     python \
     && rm -rf /var/lib/apt/lists/*
 
+# Download the init tool Tini and make it executable for use in the final image
+
+ADD https://github.com/krallin/tini/releases/download/v0.19.0/tini-static /tini
+RUN chmod u+x /tini
+
 # Enables Yarn
 RUN corepack enable
 
 RUN echo "Node: $(node --version)"
 RUN echo "Npm: $(npm --version)"
 RUN echo "Yarn: $(yarn --version)"
 
-ARG user=joplin
-
-RUN useradd --create-home --shell /bin/bash $user
+ARG user=node
```

Comment:
Why do you change the name of the user?

Reply:
ok, I saw the answer in your PR description, but still, are we sure it does not have any repercussions?
e.g. what if someone used the user `joplin` in a previous image and used a bind mount?

Reply:
This user is only used during the build stage, so it has no effect on the final image.

Reply:
Thanks for the info.

Please don't resolve a conversation. This should be done by the person who started the conversation. However, in this case I would have resolved it anyway.